src/lib/viz/index.ts

```diff
@@ -23,3 +22,6 @@ export {
 
 // Basemap helpers
 export { createGoogleMap, createMap } from './basemap';
+
+// Widgets
+export { Category } from '../widgets';
```

Comment:
IMO it doesn't make sense to export widgets from this index.ts now, as it is a separate namespace (and in the near future we will also extract the 'sources').